content/en/storage-providers/seal-workers/post-workers.md

```diff
@@ -0,0 +1,136 @@
+
+The windowPoSt process requires reading random leafs of all the sealed sectors in a proving deadline. When setting up windowPoSt workers one needs to consider how the workers can access those files. The PoSt workers can ask any other worker to read challenges for them, including the lotus-miner process, but it will prefer reading it from local paths.
+
+{{< alert icon="warning" >}}
+Storage providers should design their worker sectors access according to their setup size and also redundancy required. If only one worker has read access to the sealed sectors, it can create a single point of failure. If you have multiple partitions in a deadline, having multiple PoSt workers requesting a single worker to read challenges for them may cause unwanted I/O load.
+{{< /alert >}}
+
+### Environment variables
+
+Remember to have the appropriate Nvidia-drivers and nvidia-opencl-icd installed if running OpenCL on your worker. If using CUDA, install the CUDA-toolkit and build lotus binaries with `FFI_USE_CUDA=1`
+
+The following environment variables are required to be set before starting the worker.
+
+```
+export MINER_API_INFO:<TOKEN>:/ip4/<miner_api_address>/tcp/<port>/http
+export BELLMAN_CUSTOM_GPU="MODEL-NAME:CORES" # If you´re using a custom GPU
+export FIL_PROOFS_PARAMETER_CACHE=/fast/disk/folder # > 100GiB!
+```
+
+When the PoSt worker starts, it needs to read and verify the Filecoin proof parameters. We recommend copying them over from your lotus-miner machine. Otherwise, they will be downloaded locally on first run.
+
+The PoSt workers will fail to start if the file descriptor limit is not set high enough. This limit can be raised temporarily before starting the worker by running the command `ulimit -n 1048576`. Although, we recommend setting it permanently by following the [Permanently Setting Your ULIMIT System Value]({{< relref "kb#soft-fd-limit" >}}) guide.
+
+{{< alert icon="tip" >}}
+When fetching parameter files, remember to set the [`IPFS_GATEWAY` variable when running from China]({{< relref "nodes-in-china" >}})
+{{< /alert >}}
+
+### Run the PoSt worker
+
+```shell
+lotus-worker run <flags>
+```
+
+The above command will start the worker. You´ll need to specify which PoSt operation you want the worker to perform with one of the following flags set to true.
+
+```
+   --winningpost                 enable winning post (default: false)
+   --windowpost                  enable window post (default: false)
+```
+
+A PoSt worker instance can only be either a winningPoSt worker, or a windowPoSt worker. Enabling a PoSt tasks will automatically disable all other tasks.
+
+When a winningPoSt or windowPoSt worker connects to the _lotus-miner_, the lotus miner will delegate all winningPoSt or windowPoSt tasks to that worker. If both tasks are delegated to seperate PoSt workers, no PoSt tasks will be executed locally on the miner instance. If a worker is stopped, the lotus-miner instance switches back to local PoSt automatically.
```

Comment:
```suggestion
When a winningPoSt or windowPoSt worker connects to the `lotus-miner`, it will delegate all winningPoSt or windowPoSt tasks to that worker. If both tasks are delegated to separate PoSt workers, the `lotus-miner` will execute no PoSt tasks locally on the miner instance. If a worker is stopped, the `lotus-miner` instance automatically switches back to local PoSt.
```